### PR TITLE
Add uups proxy package

### PR DESCRIPTION
**Pull request by FuzzB0t**

### CODE REVIEW

File: src/UUPS-Proxy/InitialModuleBundle.sol [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import "./UpgradeModule.sol";
+
+// The below contract is only used during initialization as a kernel for the first release which the system can be upgraded onto.
+// Subsequent upgrades will not need this module bundle
+// In the future on live networks, we may want to find some way to hardcode the owner address here to prevent grieving
+
+// solhint-disable-next-line no-empty-blocks
+contract InitialModuleBundle is OwnerModule, UpgradeModule {
+
+}
```
Comment (mjlescano) [inline src/UUPS-Proxy/InitialModuleBundle.sol:14]: add new lines at the end of files 🙏

**Review by mjlescano — APPROVED** [review]

File: src/UUPS-Proxy/UpgradeModule.sol [hunk 2]
```diff
@@ -0,0 +1,19 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.11 <0.9.0;
+
+import "@synthetixio/core-contracts/contracts/proxy/UUPSImplementation.sol";
+import "@synthetixio/core-contracts/contracts/ownership/OwnableStorage.sol";
```
Comment (mjlescano) [inline src/UUPS-Proxy/UpgradeModule.sol:5]:
```suggestion
import {UUPSImplementation} from "@synthetixio/core-contracts/contracts/proxy/UUPSImplementation.sol";
import {OwnableStorage} from "@synthetixio/core-contracts/contracts/ownership/OwnableStorage.sol";
```

File: uups-proxy-oz.cannonfile.toml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+name = "universal-upgradeable-proxy"
+version = "<%= package.version %>"
+preset = 'oz'
+description = "Universal Proxy using OpenZeppelin UUPS contracts"
+
+[deploy.UUPSUpgradeableMock]
```
Comment (dbeal-eth) [inline uups-proxy-oz.cannonfile.toml:6]: wierd ordering not putting var in the front and deploy randomly there instead

File: package.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "contracts",
+  "version": "1.0.0",
+  "description": "Contracts for various packages published on the [Cannon Registry](https://usecannon.com/packages).",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (dbeal-eth) [inline package.json:10]: please specify the intended test script here so its easy to run

**Review by dbeal-eth — APPROVED** [review]
looking great very minor things remain.

**Pull request merged**